Bindings/Java/tests/TestEditMarkers.java

```diff
@@ -0,0 +1,65 @@
+import org.opensim.modeling.*;
+import java.io.IOException;
+import static java.nio.file.StandardOpenOption.*;
+import java.nio.file.*;
+
+class TestEditMarkers {
+  public static void main(String[] args) {
+    try {
+        // GUI model loading
+        Model model = new Model("gait10dof18musc_subject01.osim");
+        OpenSimContext context=null;
+        context = new OpenSimContext(model.initSystem(), model);
+        MarkerSet markerset = model.getMarkerSet();
+        // The following code exercises the Marker->Add New  functionality
```

Comment:
What exactly is the "Add New" callback method? The code you see here is essentially what's invoked when the GUI user picks "Add New" context command from the "Markers" node in the navigator. If you see a better way to implement the functionality, here is the perfect place to replace my implementation and then I can port it to the GUI while keeping a test case on the core side to make sure we don't break it moving forward. I have to strip the GUI code down to raw API calls since the tests don't have access to GUI classes/functions/abstractions.